### DISCUSSION TITLE
Norwegian Bokmål Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#556] [Language Leaders]
Starting discussion here for Norwegian Bokmål language leader (`nb`)

**Comment by synesthesiam** [comment]
@pckahrs would you be interested in being a language leader for Norwegian Bokmål?

**Comment by LaStrada** [comment]
If no one volunteers, I can do so 👋 I have a lot on my plate, but I can take this role for now. There haven't been much activity for Norwegian Bokmål (`nb`) and I can help getting it up to date and help others. I have currently [3 open PRs](https://github.com/home-assistant/intents/pulls/LaStrada). I'm familiar with git and use it every day.

**Reply by synesthesiam** [reply]
Thanks for volunteering! Adding you as a language leader now, @LaStrada

**Reply by andebor** [reply]
I you'd like to share the burden, I'd be happy to volunteer as a secondary to help speed up the review process. I too have [3 open PRs](https://github.com/home-assistant/intents/pulls/andebor) currently.

**Reply by LaStrada** [reply]
I've just got a newborn so I'm a bit busy as we speak! Would be nice with some help, either permanent or at least temporary.

**Reply by andebor** [reply]
Congratulations! Enjoy this time and let us worry about the PRs
